packages-proprietary/tokens/src/components/ams/table-of-contents.tokens.json

```diff
@@ -23,11 +23,6 @@
           "padding-block-start": { "value": "{ams.space.s}" },
           "padding-inline-start": { "value": "{ams.space.l}" }
         }
-      },
-      "heading": {
```

Comment:
We should probably remove the `ams-table-of-contents__heading` class as well.

Reply:
Removing it would be a breaking change, but I've deprecated it.